docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+
+```
+JIRA_BASE_URL=https://your-jira-instance.com
+SECRET_HEADER_KEY=Your-Header
+SECRET_HEADER_VALUE=abc123
+JIRA_CLIENT_ID=abc123
+JIRA_CLIENT_SECRET=abc123
+APP_BASE_URL=https://1234-123-123-12.ngrok-free.app
+APP_HOME_PAGE_URL=slack://app?team=YOUR_TEAM_ID&id=YOUR_APP_ID&tab=home
+```
+
+Start your local server:
+
+```bash
+python app.py
+```
+
+If your app is up and running, you'll see a message noting that the app is starting to receive messages from a new connection.
+
+## Setting up your workflow in Workflow Builder {#workflow}
+
+1. Within your development workspace, open Workflow Builder by clicking your workspace name and then selecting **Tools** > **Workflow Builder**. 
+2. Select **New Workflow** > **Build Workflow**.
+3. Click **Untitled Workflow** at the top of the pane to rename your workflow. We'll call it **Create Issue**. For the description, enter _Creates a new issue_, then click **Save**.
+
+![Workflow details](/img/custom-steps-jira/1.png)
+
+4. Select **Choose an event** under **Start the workflow...**, and then select **From a link in Slack**. Click **Continue**.
+
+![Start the workflow](/img/custom-steps-jira/2.png)
+
+5. Under **Then, do these things** click **Add steps** to add the custom step. Scroll down to the bottom of the list on the right-hand pane and select **Custom**, then **BoltPy Jira Functions** > **Create an issue**. Enter the project details, issue type (optional), summary (optional), and description (optional). Click **Save**.
+
+![Custom function](/img/custom-steps-jira/3.png)
+
+6. Add another step and select **Messages** > **Send a message to a channel**. Select **Channel where the workflow was used** from the drop-down list and then select **Insert a variable** and **Issue url**. Click **Save**.
+
+![Insert variable for issue URL](/img/custom-steps-jira/4.png)
+
+7. Click **Publish** to make the workflow available to your workspace.
+
+## Running your app {#run}
+
+1. Copy your workflow link.
+2. Navigate to your app's home tab and click **Connect an Account** to connect your JIRA account to the app. 
+
+![Connect account](/img/custom-steps-jira/5.png)
+
+3. Click **Allow** on the screen that appears.
+
+![Allow connection](/img/custom-steps-jira/6.png)
+
+4. In any channel, post the workflow link you copied.
+5. Click **Start Workflow** and observe as the link to a new JIRA ticket is posted in the channel. Click the link to be directed to the newly-created issue within your JIRA project.
+
+![JIRA issue](/img/custom-steps-jira/7.png)
+
+When finished, you can click the **Disconnect Account** button in the home tab to disconnect your app from your JIRA account.
+
+## Next steps {#next-steps}
+
+Congratulations! You've successfully customized your workspace with custom steps in Workflow Builder. Check out these links to take the next steps in your journey.
+
+* To learn more about Bolt for Python, refer to the [getting started](../getting-started) documentation.
```

Comment:
```suggestion
* To learn more about Bolt for Python, refer to the [getting started](/getting-started) documentation.
```

i think URL slug is the way to do this but i always get them mixed up and maybe both ways work